video2dataset/worker.py

```diff
@@ -48,7 +49,7 @@ def __init__(
         self.oom_shard_count = oom_shard_count
         self.encode_format = encode_format
         self.data_reader = VideoDataReader(video_height, video_width, timeout, find_format_timeout, max_format_tries)
-        self.subsampler = NoOpSubsampler()
+        self.clipping_subsampler = ClippingSubsampler(oom_clip_count)
```

Comment:
I suggest we branch here to use either no op either clipping (and in the future others)